nixos/modules/virtualisation/oci-containers.nix

```diff
@@ -255,7 +255,9 @@ let
           ${cfg.backend} login \
           ${container.login.registry} \
           --username ${container.login.username} \
-          --password-stdin < ${container.login.passwordFile}
+          --password-stdin < ${container.login.passwordFile} \
+          || ${cfg.backend} image inspect ${container.image} >/dev/null \ # check if image exist locally
+          || echo "image doesn't exist locally and login failed" && exit 1
```

Comment:
```suggestion
          || { echo "image doesn't exist locally and login failed"; exit 1; }
```

Reply:
we don't want a `exit 1` if the login succeeds

Reply:
my bad, updated